packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -289,6 +289,15 @@ mixin WidgetsBinding on BindingBase, SchedulerBinding, GestureBinding, RendererB
           return _forceRebuild();
         },
       );
+
+      registerServiceExtension(
+        name: 'didSendFirstFrameEvent',
+        callback: (_) async {
+          return <String, dynamic>{
+            'enabled': _needToReportFirstFrame ? 'false' : 'true'
```

Comment:
I believe this could just be `'enabled': _needToReportFirstFrame` without the ternary operator (at least according to the docs anything that's JSON-encodable is ok here).

Reply:
That is correct. However, pragmatically, I think it is best to be consistent with the existing boolean service extensions in package:flutter which use `'false'` and `'true'`.